lib/processors/lib/utils.py

```diff
@@ -11,7 +12,11 @@ def get(self):
             userid = self.get_argument('userid')
             privatekey_pem = self.get_argument('privatekey', None)
             publickey_pem = self.get_argument('publickey', None)
-            user = User(userid, publickey_pem=publickey_pem, privatekey_pem=privatekey_pem)
+            user_blob = get_user(userid)
```

Comment:
note: i mean specifically a db connection to the `quantifiedSelf` db... i think processors can make a rethink connection to their own db if they want to.